Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased]
 
+### Changed
+- UI toolkit now uses the "UI" action map of project-wide actions as their default input actions. Removing bindings or renaming actions will break UI input for UI toolkit.
```

Comment:
I think this is expected given the solution we have previously discussed but its unfortunate since the action is basically required (driven by the implementation). In the long run I believe we should consider design changes that would allow the integration to own the action while the editor would allow reconfiguring it.

Reply:
For the CHANGED message I believe it could be more clear to users if you added the delta to the description, e.g. "UI toolkit now uses the "UI" action map of project-wide actions as their default input actions instead of..." basically highlighting the switch from hard-coded to customisable.